lisa/tools/kernel_config.py

```diff
@@ -53,3 +58,19 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
             self.config_path = f"/usr/boot/config-{kernel_ver}"
         else:
             self.config_path = f"/boot/config-{kernel_ver}"
+
+
+class KernelConfigFreeBSD(KernelConfig):
+    def is_built_in(self, config_name: str) -> bool:
+        return self.node.tools[KLDStat].module_statically_linked(config_name)
+
+    def is_built_as_module(self, config_name: str) -> bool:
```

Comment:
This method is exactly same as KernelConfig. So it doesn't need to be inherited.

Reply:
I made an error while fixing earlier comment. It should be fixed now